R/priorsense.R

```diff
@@ -51,13 +51,19 @@ create_priorsense_data.brmsfit <- function(x, ...) {
 
 #' @exportS3Method priorsense::log_lik_draws
 log_lik_draws.brmsfit <- function(x, ...) {
+  nchains <- nchains(x)
+  niters <- niterations(x)
   log_lik <- log_lik(x, ...)
-  log_lik <- posterior::as_draws_array(log_lik)
+  nobs <- length(log_lik) / (nchains * niters)
```

Comment:
`nobs` is always the number of columns of `log_lik` so no need to infer that from length, nchains, and niters.

Reply:
Of course! That's much simpler